src/test/java/pl/nextleveldev/smart_route/route/api/RouteControllerTest.java

```diff
@@ -0,0 +1,78 @@
+package pl.nextleveldev.smart_route.route.api;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import pl.nextleveldev.smart_route.route.RouteService;
+
+import java.util.Collections;
+import java.util.List;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+import static org.assertj.core.api.AssertionsForClassTypes.assertThatThrownBy;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.when;
+
+@ExtendWith(MockitoExtension.class)
+class RouteControllerTest {
+
+    public static final String START_LOCATION = "Marszałkowska 10";
+    public static final String DESTINATION = "Wałbrzyska 7";
+    public static final String API_ERROR = "API error";
```

Comment:
The “API error” is very general and does not give information what went wrong.

Depending on the cause of the occurrence of the error
```suggestion
    public static final String API_ERROR = "An error occurred while communicating with the external API";
    public static final String API_ERROR = "Unexpected error in API processing";
    // ...
```